src/test/ui-test/install.test.ts

```diff
@@ -0,0 +1,238 @@
+import { expect } from 'chai';
+import * as fs from 'fs-extra';
+import { repeat, TimeoutError } from '@theia-extension-tester/repeat';
+import {
+    EditorView,
+    error,
+    ExtensionsViewItem,
+    ExtensionsViewSection,
+    QuickOpenBox,
+    SideBarView,
+    ViewControl
+} from 'vscode-extension-tester';
+import { Workbench } from 'vscode-uitests-tooling';
+
+describe('Marketplace presentation', function () {
+    this.timeout(30000);
+    this.slow(10000);
+
+    let extensionMetadata: { [key: string]: any };
+    let name: string;
+    let marketplace: ExtensionsViewSection;
+
+    before('Load metadata', function () {
+        extensionMetadata = fs.readJsonSync('package.json', {
+            encoding: 'utf-8'
+        });
+        name = extensionMetadata['displayName'];
+    });
+
+    beforeEach('Open marketplace', async function () {
+        marketplace = await openMarketplace(this.timeout() - 1000);
+    });
+
+    afterEach('Close marketplace', async function () {
+        await closeMarketplace(this.timeout() - 1000);
+    });
+
+    it('Has correct title', async function () {
+        const [_, extension] = await openExtensionPage(marketplace, name, this.timeout() - 1000);
+        expect(await extension.getTitle()).to.be.equal(extensionMetadata['displayName']);
+    });
```

Comment:
optional: better format experience could be with empty row between each `it` functions